tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -450,5 +450,6 @@ TEST_F(FileTest, ComputeByteAddressBufferLoad) {
 TEST_F(FileTest, ComputeByteAddressBufferStore) {
   runFileTest("compute.byte-address-buffer.store.hlsl");
 }
+TEST_F(FileTest, ComputeBuffer) { runFileTest("compute.buffer.hlsl"); }
```

Comment:
This belongs to the `type` category. Preferably should be named as `type.buffer.hlsl`